public/locales/bg/gamepage.json

```diff
@@ -66,14 +64,6 @@
         "moving": "Преместване",
         "repairing": "Поправяне"
     },
-    "info": {
```

Comment:
those removals doesnt seems right

Reply:
that's what I was thinking, but that's what happens when I run yarn i18next, even on the up-to-date main branch

Reply:
`yarn i18next` or `yarn i18n`? should be the same, I guess. But I will try to investigate this later.